src/components/Register/Register.jsx

```diff
@@ -47,75 +48,6 @@ const Register = () => {
         validate={validate}
         render={({ handleSubmit }) => (
           <form onSubmit={handleSubmit} className="registration-form__form">
-            <div className="registration-form__field">
-              <label className="registration-form__label">User Name:*</label>
-              <Field
-                type="text"
-                name="userName"
-                component="input"
-                className="registration-form__input"
-              />
-              <div className="registration-form__error">
-                <Field
-                  name="userName"
-                  subscription={{ error: true, touched: true }}
-                >
-                  {({ meta: { touched, error } }) =>
-                    touched && error ? (
-                      <span className="registration-form__error-text">
-                        {error}
-                      </span>
-                    ) : null
-                  }
-                </Field>
-              </div>
-            </div>
-            <div className="registration-form__field">
-              <label className="registration-form__label">First Name:*</label>
-              <Field
-                type="text"
-                name="firstName"
-                component="input"
-                className="registration-form__input"
-              />
-              <div className="registration-form__error">
-                <Field
-                  name="firstName"
-                  subscription={{ error: true, touched: true }}
-                >
-                  {({ meta: { touched, error } }) =>
-                    touched && error ? (
-                      <span className="registration-form__error-text">
-                        {error}
-                      </span>
-                    ) : null
-                  }
-                </Field>
-              </div>
-            </div>
-            <div className="registration-form__field">
-              <label className="registration-form__label">Last Name:*</label>
-              <Field
-                type="text"
-                name="lastName"
-                component="input"
-                className="registration-form__input"
-              />
-              <div className="registration-form__error">
-                <Field
-                  name="lastName"
-                  subscription={{ error: true, touched: true }}
-                >
-                  {({ meta: { touched, error } }) =>
-                    touched && error ? (
-                      <span className="registration-form__error-text">
-                        {error}
-                      </span>
-                    ) : null
-                  }
-                </Field>
-              </div>
-            </div>
```

Comment:
Why did we remove the form?

Reply:
If it's for testing then the PR wasn't ready for review. Please mark it as draft and complete all the necessary features then ask for review. Thanks

Reply:
ok vaiya